sass/components/_on-this-page.scss

```
@@ -0,0 +1,36 @@
+/// internal page navigation
+/// i.e., the "On This Page" section
+
+
+.on-this-page {
+  $h-padding: 12px;
+  font-size: 0.9em;
+  padding-block: calc($h-padding/2);
+  ul {
+    list-style-position: outside;
+    list-style-type: none;
+    padding-inline-start: $h-padding;
+  }
+  > ul {
+    padding-inline: $h-padding;
+  }
+  li[data-active=true] {
+    font-weight: bolder;
+    li {
+        font-weight: normal;
+    }
+  }
+  a {
+    @media ($bp-tablet-portrait-down) {
+        // increase hit area for touchscreens
+        padding-block: 3px;
+    }
+    display: block;
+    // copied from .media-content
+    text-decoration: none;
+    color: $link-color;
+    &:hover {
+      text-shadow: 0 0 0.9px $link-hover-shadow-color, 0 0 0.9px $link-hover-shadow-color;
+    }
+  }
+}
```

https://github.com/bevyengine/bevy-website/assets/188612/563020aa-03b9-4632-ac0e-f72eb203c07a

Changed:
- `.on-this-page { font-size }`
- `a { padding, text-wrap }`

```suggestion
.on-this-page {
  $h-padding: 12px;
  font-size: 0.95em;
  padding-block: calc($h-padding/2);
  ul {
    list-style-position: outside;
    list-style-type: none;
    padding-inline-start: $h-padding;
  }
  > ul {
    padding-inline: $h-padding;
  }
  li[data-active=true] {
    font-weight: bolder;
    li {
        font-weight: normal;
    }
  }
  a {
    @media ($bp-tablet-portrait-down) {
        // increase hit area for touchscreens
        padding-block: 3px;
    }
    display: block;
    // copied from .media-content
    text-decoration: none;
    color: $link-color;
    padding: 2px 0;
    text-wrap: balance;
    &:hover {
      text-shadow: 0 0 0.9px $link-hover-shadow-color, 0 0 0.9px $link-hover-shadow-color;
    }
  }
}
```

---

I deleted the font-size rule and added text-wrap. I already added extra padding for mobile on line 26

---

I was suggesting it also for desktop, for me it's too dense otherwise. But it's OK.

---

I see. I don't think destkop needs it.

---

gave it 1px